### PR TITLE
Replace unnecessary scope(success)

### PR DESCRIPTION
**Pull request by AndrejMitrovic**
@TrustHenry this should make things clearer.

### CODE REVIEW

**Review by TrustHenry — APPROVED** [review]
LGTM

File: source/agora/network/NetworkManager.d [hunk 1]
```diff
@@ -410,9 +409,6 @@ public class NetworkManager
 
         while (!this.peerLimitReached())
         {
-            scope (success)
-                this.taskman.wait(this.node_config.retry_delay.msecs);
```
Comment (Geod24) [inline source/agora/network/NetworkManager.d]: There's a small difference here: It won't sleep after the `continue` on L412. But I'm wondering, isn't there an infinite loop there ?

File: source/agora/network/NetworkManager.d [hunk 2]
```diff
@@ -269,9 +269,6 @@ public class NetworkManager
         {
             while (1)
             {
-                scope (success)
-                    this.outer.taskman.wait(1.seconds);
```
Comment (Geod24) [inline source/agora/network/NetworkManager.d]: Same here

File: source/agora/network/NetworkManager.d (hunk 1 above)
Comment (AndrejMitrovic) [reply source/agora/network/NetworkManager.d]: Yes until `peerLimitReached()` is true.

Comment (Geod24) [reply source/agora/network/NetworkManager.d]: But it's a busy wait, before it wasn't

Comment (AndrejMitrovic) [reply source/agora/network/NetworkManager.d]: It's the same as before?

https://github.com/bpfkorea/agora/blob/aef86ff0151c36ff83e9af9d05ec77fc7b75fd65/source/agora/network/NetworkManager.d#L208-L212

**Comment by AndrejMitrovic** [comment]
I had to revert two changes. I missed there was still a `continue` in there. Maybe this can be simplified in the future..

**Review by Geod24 — APPROVED** [review]

**Pull request merged**